source/wp-content/themes/wporg-parent-2021/sass/block-styles.scss

```diff
@@ -121,3 +121,51 @@
 		}
 	}
 }
+
+// A heading with an arrow and other content.
+.heading-with-arrow {
+	transition: background-color 0.15s linear;
+
+	figure {
+		transition: all 0.15s linear;
+	}
+
+	a {
+		color: var(--wp--preset--color--white) !important;
+		text-underline-offset: 2px;
+		cursor: pointer;
```

Comment:
Are these necessary? If we want to offset the underline, we should probably do that for all links.